api/src/main/scala/hmda/api/HmdaApi.scala

```diff
@@ -36,9 +41,27 @@ object HmdaApi
 
   //Start up API Actors
 
-  createSingleLarValidator(system)
+  val supervisor = HmdaSupervisor.createSupervisor(system)
+
+  (supervisor ? FindActorByName(SingleLarValidation.name))
```

Comment:
I remember you had mentioned that the actors would time out after a certain period of inactivity.  If that's the case, would we still want to start up all the actors right away (when they might be killed later), or should we wait until they receive activity before spinning them up?